app/src/interfaces/file/file.vue

```diff
@@ -46,7 +46,7 @@
 
 			<v-list>
 				<template v-if="file">
-					<v-list-item :download="file.filename_download" :href="assetURL">
+					<v-list-item clickable :href="addTokenToURL(getRootPath() + `assets/${file.id}?download`)">
```

Comment:
I believe we should still keep the `download` prop here, since it's meant to be passed down to the `<a>` tag/component in `<v-list-item>` for the download filename: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download as the "Filename (Download)" field is customizable in each file.

Reply:
Yes, I had also thought at first, but then I looked in the files-interface and took this logic, maybe this must be adjusted there?

Reply:
download name is added in interface file

Reply:
> Yes, I had also thought at first, but then I looked in the files-interface and took this logic, maybe this must be adjusted there?

Makes sense! Let's add the download prop in `files` interface as well. Likely was just a missed detail when adding download link to `files` interface in ##12220.